defaults-o2-dev-fairroot.sh

```diff
@@ -29,14 +27,25 @@ overrides:
       which gfortran || { echo "gfortran missing"; exit 1; }
       which cc && test -f $(dirname $(which cc))/c++ && printf "#define GCCVER ((__GNUC__ << 16)+(__GNUC_MINOR__ << 8)+(__GNUC_PATCHLEVEL__))\n#if (GCCVER < 0x060200)\n#error \"System's GCC cannot be used: we need at least GCC 6.X. We are going to compile our own version.\"\n#endif\n" | cc -xc++ - -c -o /dev/null
   ROOT:
-    tag: "v6-08-02"
+    version: "v6-08-06+git_%(short_hash)s"
+    tag: e2951b4cc49ee88651a69a0b54a583333c0ae263
     source: https://github.com/root-mirror/root
     requires:
       - GSL
       - opengl:(?!osx)
       - Xdevel:(?!osx)
       - FreeType:(?!osx)
       - Python-modules
+      - GCC-Toolchain:(?!osx)
+  AliRoot:
+    requires:
```

Comment:
AliRoot dependencies for O2 must include ZeroMQ, which is not listed in the default `aliroot.sh` recipe. Does this answer your question?